Bpi/Sdk/Bpi.php

```diff
@@ -94,7 +94,7 @@ public function push(array $data)
         $nodes->firstItem('name', 'node')
             ->template('push')
             ->eachField(function ($field) use ($data) {
-                if (!empty($data[(string)$field]))
+                if (!isset($data[(string)$field])) // nb: check on empty() produce errors.
```

Comment:
I agree with @guddo here. If possible please put comments on a separate line.

In general I do not think the BPI client which should not be tied to Drupal specifically must follow Drupal coding standards.